helm/prometheus-agent/charts/prometheus-agent/values.yaml

```diff
@@ -8,6 +8,28 @@ image:
   sha: ""
   pullPolicy: "Always"
 
+affinity:
+  nodeAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Why is this not indented ? Also, i'd probably only have affinity like Loki or tempo do